internal-tooling/performanceUtils.ts

```diff
@@ -224,3 +225,58 @@ export function getValidationType(): 'md5' | 'crc32c' | boolean | undefined {
     return 'md5';
   }
 }
+
+/**
+ * Converts the supplied test results from javascript objects to cloud monitoring format.
+ *
+ * @param {TestResult[]} results array of test iteration results.
+ * @param {string} bucket the bucket name used for the test
+ *
+ * @returns {AsyncGenerator<string>} a string containing the results of the conversion.
+ */
+export async function* convertToCloudMonitoringFormat(
+  results: TestResult[],
+  bucketName: string
+): AsyncGenerator<string> {
+  for (const curResult of results) {
+    const throughput =
+      curResult.objectSize >= SSB_SIZE_THRESHOLD_BYTES
```

Comment:
can you add a comment explaining the unit conversion happening here?

Reply:
Added.